discord-scripts/role-management.ts

```diff
@@ -0,0 +1,34 @@
+import { Client } from "discord.js"
+
+export default async function manageRole(
+  discordClient: Client,
+  guildId: string,
+  memberId: string,
+  roleId: string,
```

Comment:
I'm not convinced we'll find ourselves wanting to do this without having handles to at least the guild and maybe member objects already... For example, the first use we're pondering, which would be in reaction to the `guildMemberAdd` event, [would receive a `GuildMember` already](https://discord.js.org/docs/packages/discord.js/14.14.1/Client:Class#guildMemberAdd)—allowing us to bypass most of the steps here.